evergreen/config.yml

```diff
@@ -33,83 +33,134 @@ functions:
           set -o errexit
           git submodule update --init --recursive
 
-tasks:
-- name: compile
-  commands:
-  - command: shell.exec
-    params:
-      working_dir: realm-core
-      shell: bash
-      script: |-
-        set -o errexit
-        set -o verbose
+  "compile":
```

Comment:
This turns the compile task into a function so that we can easily run a compile from any task where we need it.